### PR TITLE
Add secret baseline file

### PR DESCRIPTION
**Pull request by vmatyusGitHub**
## Checklist

- [x] Tick to sign-off your agreement to the [Developer Certificate of Origin (DCO) 1.1](../blob/master/DCO1.1.txt)
- [x] Added tests for code changes _or_ test/build only changes
- [x] Updated the change log file (`CHANGES.md`|`CHANGELOG.md`) _or_ test/build only changes
- [x] Completed the PR template below:

## Description

Add `.secrets.baseline`

## Schema & API Changes

- "No change"

## Security and Privacy

- "No change"

## Testing

- "No change"

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Unreleased
+- [NEW] Add Detect Secrets baseline file to scan changes for secrets in code.
```
Comment (ricellis) [inline CHANGES.md:2]: We can drop this CHANGES entry it is not relevant for users.

**Review by ricellis — APPROVED** [review]
+1, but drop the CHANGES entry

File: CHANGES.md (hunk 1 above)
Comment (vmatyusGitHub) [reply CHANGES.md]: Removed it in 7e3d03b

**Comment by vmatyusGitHub** [comment]
Thank you for reviewing! I am about to merge this.

**Pull request merged**